static/app/views/settings/organizationMembers/inviteBanner.tsx

```diff
@@ -124,13 +142,27 @@ export function InviteBanner({
       return;
     }
     setSendingInvite(true);
-    await onSendInvite(email);
+    try {
+      await api.requestPromise(
+        `/organizations/${organization.slug}/members/?referrer=github_nudge_invite`,
+        {
+          method: 'POST',
+          data: {email},
+        }
+      );
+      addSuccessMessage(tct('Sent invite to [email]', {email}));
+      onSendInvite();
+      const updatedMissingMembers = missingMembers.filter(
+        member => member.email !== email
+      );
+      setMissingMembers(updatedMissingMembers);
+    } catch {
+      addErrorMessage(t('Error sending invite'));
+    }
     setSendingInvite(false);
   };
 
-  const users = missingMembers.users;
-
-  const cards = users.slice(0, MAX_MEMBERS_TO_SHOW).map(member => {
+  const cards = missingMembers.slice(0, MAX_MEMBERS_TO_SHOW).map(member => {
```

Comment:
🤔 

hmmm... i think we _don't_ want to have the subrender components within existing components.

can we pop this out into it's own component at the end of the file?

Reply:
Also means you don't need to push the `SeeMoreCards` at the end there.
Just have that be a part of the component